### DISCUSSION TITLE
Add "External" arrow on all external links, not only sidebar

### DISCUSSION THREAD

**Discussion by miketheman** [pradyunsg/furo#775] [Ideas] [answered]
Hi there!

I'm finding the desire to have the little `external` arrow for inline docs, not only the sidebar.

https://github.com/pradyunsg/furo/blob/01887051504bbec32e241af9cebcf5cd10f656d1/src/furo/assets/styles/components/_sidebar.sass#L139-L144

It'd be great to have the ability to discern if the link I'm about to click will remain within a docs site, or if it'll take me away from it.

Would you consider a change to the lower-level CSS that uses this code more liberally?

If so, can you point me where that might live? Happy to submit a PR.

**Comment by pradyunsg** [comment, marked as answer]
I'd suggest [injecting your own CSS](https://pradyunsg.me/furo/customisation/injecting/#css-or-js) to achieve this with the code living next to your documentation itself.

I'm wary of adding knobs for functionality like this since I don't think this would be something that everyone wants and I'm biased toward a smaller surface to support here. 😅
